packages/@azure/servicebus/data-plane/test/testUtils.ts

```diff
@@ -39,104 +42,305 @@ export enum ClientType {
   TopicFilterTestDefaultSubscription,
   TopicFilterTestSubscription
 }
+const defaultLockDuration = "PT30S"; // 30 seconds in ISO 8601 FORMAT - equivalent to "P0Y0M0DT0H0M30S"
 
-export function getSenderClient(
-  namespace: Namespace,
-  clientType: ClientType
-): QueueClient | TopicClient {
-  switch (clientType) {
-    case ClientType.PartitionedQueue:
-      return namespace.createQueueClient(process.env.QUEUE_NAME || "partitioned-queue");
-    case ClientType.PartitionedTopic:
-      return namespace.createTopicClient(process.env.TOPIC_NAME || "partitioned-topic");
-    case ClientType.UnpartitionedQueue:
-      return namespace.createQueueClient(
-        process.env.QUEUE_NAME_NO_PARTITION || "unpartitioned-queue"
-      );
-    case ClientType.UnpartitionedTopic:
-      return namespace.createTopicClient(
-        process.env.TOPIC_NAME_NO_PARTITION || "unpartitioned-topic"
-      );
-    case ClientType.PartitionedQueueWithSessions:
-      return namespace.createQueueClient(
-        process.env.QUEUE_NAME_SESSION || "partitioned-queue-sessions"
+async function getEnvVars(): Promise<any> {
```

Comment:
There is nothing async going on inside `getEnvVars`
Therefore, this need not be marked as `async`
Neither does it have to return a Promise.
Have return type as `{[key: string]: string}`